README.md

```diff
@@ -306,4 +318,8 @@ outils:
 
 [Portail des sites web des équipes par PM-ROBOTIX](https://www.pm-robotix.eu/sites-de-la-coupe-et-des-equipes/)
 
-[Elements of Robotics, Mordechai Ben-Ari, Francesco Mondada, 2018, Open Access](https://www.springer.com/gp/book/9783319625324)
\ No newline at end of file
+[Forum Robot Maker](https://www.robot-maker.com/forum/)
+
+[Elements of Robotics, Mordechai Ben-Ari, Francesco Mondada, 2018, Open Access](https://www.springer.com/gp/book/9783319625324)
+
+[Robot de recherche Facebook](https://pyrobot.org/)
```

Comment:
Description plus détaillé: "PyRobot - light weight, high-level interface which provides hardware independent APIs for robotic manipulation and navigation by facebook research."